cmd/client-s3.go

```diff
@@ -54,6 +56,20 @@ const (
 	googleHostName = "storage.googleapis.com"
 )
 
+// getNumMultiPartThreads gets the number of multipart threads to be used for mc cp/mirror functions
+// If not set, default to 4.
+func getNumMultiPartThreads() uint {
+	multipartThreadStr := os.Getenv("MC_MULTIPART_THREADS")
+	if multipartThreadStr != "" {
+		numThreads, err := strconv.ParseUint(multipartThreadStr, 10, 32)
+		if err != nil {
+			log.Fatal("MC_MULTIPART_THREADS must be a positive integer.")
+		}
+		return uint(numThreads)
+	}
+	return 4
```

Comment:
Can you add a new global constant definition ? probably `const defaultMultipartThreadsNum = 4`

Reply:
done.